.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
python should have 4 spaces for indents... it should probably be specified in a specific subsection so that our JSON files (CI config, etc) are still 2 space indents.